apps/tomcat.go

```diff
@@ -165,7 +165,7 @@ type SystemLoggingReceiverTomcat struct {
 func (r SystemLoggingReceiverTomcat) Components(tag string) []fluentbit.Component {
 	if len(r.IncludePaths) == 0 {
 		r.IncludePaths = []string{
-			"/opt/tomcat/logs/catalina.out",
+			"/var/log/tomcat9/catalina.out",
```

Comment:
What impact does this change have? Looks like we are changing the default log file path. Does any distro use `/opt/tomcat/logs/catalina.out` instead? If so, should we include both?

Reply:
I'll Add both, it seems the package manager uses what I changed it to and I'm seeing more and more the `/var/log` but I've seen both and tomcat doesn't explicitly call it out in the docs that I can tell so I'll add both.

Reply:
What about tomcat10? Or tomcat8? I think we're going to need multiple file paths to cover different versions and distros.